events/0116_gc_intro-coding.html

```diff
@@ -0,0 +1,165 @@
+                    <h3 class="event__title box--small green"><small><span class="blue">Intro to Ruby Workshop</span><br />4-Session Workshop</small></h3>
+                    <h3 class="event__title"><small>On Tuesdays starting 2/2 @ Nutshell</small></h3>
+                    <p><small>6:00pm - 9pm &nbsp; | &nbsp; 212 S. 5th Ave</small></p>
+                   <aside class="notes"></aside>
+                </div>
+   -->         </section>
+
+            <section>
+                <h3>Community Events</h3>
+                <ul class="box copy--xsmall left" style="width: 48%">
+                    <li><strong>A2 Coffee House Coders</strong><br /><small>Wednesday 1/13 @ Espresso Royale (State)</small></li>
+                    <li><strong>A2 CocoaHeads - Nobody is using App Transport Security; what's next</strong><br /><small>Thursday 1/14 @ Arbormoon</small></li>
+                    <li><strong><span class="blue">Grand Circus Detroit - Class: Co.Starters</span></strong><br /><small>Tuesday, 1/19 @ Grand Circus Detroit</small></li>
+                    <li><strong>Ladies That UX Detroit</strong><br /><small>Tuesday, 1/19 @ Grand Circus Detroit</small></li>
+                    <li><strong>A2 New Tech Monthly Meetup</strong><br /><small>Tuesday 1/19 @ Law School, South Hall, Room 1225</small></li>
+                    <li><strong><span class="blue">Grand Circus Detroit - Class: Facebook and Instagram Ads</span></strong><br /><small>Tuesday, 1/26 @ Grand Circus Detroit</small></li>
+                </ul>
+                <ul class="box copy--xsmall right" style="width: 48%">
+                    <li><strong>eCommerce Ann Arbor - Making Marketing Automation Easy</strong><br /><small>Tuesday, 1/26 @ Human Element</small></li>
+                    <li><strong>WordPress Ann Arbor - Business Development for WP Freelancers</strong><br /><small>Wednesday, 1/27 @ U-M, LSA</small></li>
+                    <li><strong>Fanzoo - Learn Something Monthly Workshop</strong><br /><small>Wednesday, 1/27 @ Fanzoo</small></li>
+                    <li><strong>Pillar PluggedIn - Building Webservers on the Raspberry Pi 2</strong><br /><small>Thursday, 1/28 @ Pillar Forge</small></li>
+                    <li class="green"><em>Plus soooo many more...</em></li>
+                </ul>
+                <p class="clear copy--small"><a href="http://bit.ly/gdiaa-community">bit.ly/gdiaa-community</a></p>
+                <aside class="notes"></aside>
+            </section>
+
+            <!-- Sponsors -->
+            <section>
+                <h3>Sponsors!</h3>
+                <img src="images/sponsors-2015.3.png" class="img--bare box--small" />
+            </section>
+
+            <!-- Sponsor Spotlight -->
+            <section>
+                <h3>Sponsor Spotlight!</h3>
+                <img src="images/barracuda.jpg" class="img--bare box" width="400" />
+                <img src="images/tek-systems.jpg" class="img--bare box" width="400" />
+            </section>
```

Comment:
@kldickenson let's break this out into 2 slides with 1 logo each. Barracuda first, then Tek Systems